src/utils/joiSchema.js

```diff
@@ -0,0 +1,8 @@
+const Joi = require("joi");
+
+const bookmarkSchema = Joi.object({
+  title: Joi.string().max(255),
```

Comment:
This form lacks an ID, and other fields—which should be required—are not indicated as such.

Reply:
We can't add Id here as it is not a property being passed by client but sometime we generated during creation of the bookmark, also we validate it while we delete bookmark with id separately so instead of creating a different schema just for id I added it directly while fetching and deleting bookmark by id.